codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/protocol/HttpBindingProtocolGenerator.kt

```diff
@@ -249,6 +249,7 @@ abstract class HttpBindingProtocolGenerator : ProtocolGenerator {
                 if (op.isInputEventStream(ctx.model)) {
                     val eventStreamSerializeFn = eventStreamRequestHandler(ctx, op)
                     writer.write("builder.body = #T(context, input)", eventStreamSerializeFn)
+                    renderContentTypeHeader(ctx, op, writer, resolver)
```

Comment:
Question: This doesn't break anything for the other protocols?

Reply:
not that I've seen. we have e2e tests for event streams in aws-sdk-kotlin